lib/sidekiq/worker_killer.rb

```diff
@@ -44,6 +64,7 @@ def request_shutdown
   def shutdown
     warn "sending #{quiet_signal} to #{identity}"
     signal(quiet_signal, pid)
+    sleep(5) # gives Sidekiq API 5 seconds to update ProcessSet
```

Comment:
As a reference https://github.com/mperham/sidekiq/wiki/API#processes 
> Sidekiq::ProcessSet gets you access to near real-time (updated every 5 sec)

This is to cover an issue where the process was "quieted" although, `return false if process["busy"] != 0 && process["quiet"] == "true"` didn't get the up-to-date information from the API and it shut down without grace_time.

Reply:
https://github.com/klaxit/sidekiq-worker-killer/pull/10